src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -601,6 +601,7 @@ public void expire()
             if (logger.isDebugEnabled() && getConference().includeInStatistics())
             {
                 logger.debug(transceiver.getNodeStats().prettyPrint(0));
+                logger.debug(bitrateController.getDebugState().toJSONString());
```

Comment:
Why isn't this method just using `Endpoint.getDebugState()` here anyway?

Reply:
One unfortunate thing is that JSONObject doesn't support pretty-printing, so unfortunately the output is much less friendly to read in the logs.